Gifski/AppDelegate.swift

```diff
@@ -130,3 +122,40 @@ extension AppDelegate {
 		mainWindowController.convert(url)
 	}
 }
+
+class LaunchCompletions {
```

Comment:
Would also be great to have some doc comments for this with an example on how to use it.